### DISCUSSION TITLE
Space Delivery

### DISCUSSION THREAD

**Discussion by giscus** [aduros/wasm4#383] [Games]
# Space Delivery

Build retro games using WebAssembly for a fantasy console

https://wasm4.org/play/space-delivery/

**Comment by kirinokirino** [comment]
Description from the https://kirinokirino.itch.io/space-delivery

Explore the space at your own leisure. If you'd like, you can find and land on a specific planet. The bottom line shows the distance to that planet, you probably would need to fly around a bit to find where it is. 

Arrows to move the ship (it changes the shape depending on its speed)

If you get stuck in a planets gravity well, use booster packs with X or SPACEBAR.

To use auto-stabilizers press  Z, but notice that their power is limited and they wouldn't help in a strong gravity field.

You can shoot particles with the left mouse click, but they don't do anything, this is a peaceful game.

**Reply by CanyonTurtle** [reply]
This is a really creative game!
